### ISSUE TITLE
formatVCF returns VCF with all POS = 1

### ISSUE THREAD

**Issue by jcm6t** [zwdzwd/sesame#31]
Sesame 1.8.7

Here is sample code for our 458 arrays. We've also output some objects to look at their format. Note the warnings()

```
annoS<-sesameDataPullVariantAnno_SNP('EPIC','hg38')
Retrieving SNP annotation from  https://zwdzwd.s3.amazonaws.com/InfiniumAnnotation/20200704/EPIC/EPIC.hg38.snp_overlap_b151.rds ... Done.

class(annoS)
[1] "GRanges"
attr(,"package")
[1] "GenomicRanges"

head(annoS)
GRanges object with 6 ranges and 5 metadata columns:
             seqnames    ranges strand |          rs  designType           U         REF         ALT
                <Rle> <IRanges>  <Rle> | <character> <character> <character> <character> <character>
   rs3936238     chr1   4071666      - |   rs3936238           I         REF           A           G
    rs877309     chr1  11429621      - |    rs877309          II         REF           A           G
  rs11249206     chr1  24951491      + |  rs11249206          II         ALT           C           T
    rs715359     chr1 177462032      + |    rs715359           I         REF           T           C
   rs2804694     chr1 181362697      - |   rs2804694          II         ALT           G           A
    rs966367     chr2  12008094      + |    rs966367          II         ALT           C           T
  -------
  seqinfo: 26 sequences from an unspecified genome; no seqlengths

annoI<-sesameDataPullVariantAnno_InfiniumI('EPIC','hg38')
Retrieving SNP annotation from  https://zwdzwd.s3.amazonaws.com/InfiniumAnnotation/20200704/EPIC/EPIC.hg38.typeI_overlap_b151.rds ... Done.

class(annoI)
[1] "GRanges"
attr(,"package")
[1] "GenomicRanges"

head(annoI)
GRanges object with 6 ranges and 5 metadata columns:
             seqnames    ranges strand |  designType     In.band         REF         ALT          rs
                <Rle> <IRanges>  <Rle> | <character> <character> <character> <character> <character>
  cg19411932     chr1    827749      + |           I         REF           C           T rs534577547
  cg16578267     chr1   1001667      - |           I         REF           C           G rs577785168
  cg17801765     chr1   1022896      + |           I         REF           G           C rs113543998
  cg18148498     chr1   1445240      - |           I         REF           A           G   rs1153094
  cg12133962     chr1   3173572      + |           I         REF           T           C   rs3795267
  cg24306648     chr1   3638204      - |           I         REF           G           A  rs58651937
  -------
  seqinfo: 26 sequences from an unspecified genome; no seqlengths

vcflist<-lapply(ssets,formatVCF, annoS=annoS,annoI=annoI)
There were 50 or more warnings (use warnings() to see the first 50)


warnings()
Warning messages:
1: In cbind(as.character(GenomicRanges::seqnames(annoI)),  ... :
  number of rows of result is not a multiple of vector length (arg 2)
2: In cbind(as.character(GenomicRanges::seqnames(annoI)),  ... :
  number of rows of result is not a multiple of vector length (arg 2)
3: In cbind(as.character(GenomicRanges::seqnames(annoI)),  ... :
  number of rows of result is not a multiple of vector length (arg 2)
4: In cbind(as.character(GenomicRanges::seqnames(annoI)),  ... :
  number of rows of result is not a multiple of vector length (arg 2)
snipped....

dim(vcflist[[1]])
[1] 865   8

head(vcflist[[1]],20)
           #CHROM POS         ID REF ALT QUAL FILTER                   INFO
rs877309     chr1   1   rs877309   A   G   73   PASS PVF=0.927;GT=1/1;GS=73
rs715359     chr1   1   rs715359   T   C   92   PASS PVF=0.966;GT=1/1;GS=92
cg16578267   chr1   1 cg16578267   C   G   73   PASS PVF=0.083;GT=0/0;GS=73
cg18148498   chr1   1 cg18148498   A   G   83   PASS PVF=0.051;GT=0/0;GS=83
cg24306648   chr1   1 cg24306648   G   A   54   PASS PVF=0.113;GT=0/0;GS=54
cg08033730   chr1   1 cg08033730   A   G   73   PASS PVF=0.068;GT=0/0;GS=73
cg10710472   chr1   1 cg10710472   C   T   73   PASS PVF=0.078;GT=0/0;GS=73
cg09118017   chr1   1 cg09118017   G   C   73   PASS PVF=0.075;GT=0/0;GS=73
cg01882033   chr1   1 cg01882033   C   G   63   PASS PVF=0.091;GT=0/0;GS=63
cg10952925   chr1   1 cg10952925   G   A   73   PASS PVF=0.067;GT=0/0;GS=73
cg16063783   chr1   1 cg16063783   C   T   73   PASS PVF=0.081;GT=0/0;GS=73
cg22367705   chr1   1 cg22367705   A   G   35   PASS PVF=0.186;GT=0/0;GS=35
cg10832476   chr1   1 cg10832476   G   T   54   PASS PVF=0.130;GT=0/0;GS=54
cg23844904   chr1   1 cg23844904   A   G   63   PASS PVF=0.089;GT=0/0;GS=63
cg21680327   chr1   1 cg21680327   G   C   54   PASS PVF=0.121;GT=0/0;GS=54
cg08673396   chr1   1 cg08673396   G   C   83   PASS PVF=0.056;GT=0/0;GS=83
cg00148881   chr1   1 cg00148881   T   C   83   PASS PVF=0.062;GT=0/0;GS=83
cg01134183   chr1   1 cg01134183   C   A   63   PASS PVF=0.099;GT=0/0;GS=63
cg25979624   chr1   1 cg25979624   C   T    4   FAIL  PVF=0.285;GT=0/1;GS=4
cg20680664   chr1   1 cg20680664   G   A   63   PASS PVF=0.091;GT=0/0;GS=63

```

**Comment by zwdzwd** [comment]
Thanks for the feedback. This is a bug from namespace conflict between GenomicRanges::end and stats::end. The fix has been uploaded and will be part of 1.8.8 and 1.9.16. Here is a temporary link
https://www.dropbox.com/s/e1n3x27h3zkfyat/sesame_1.8.8.tar.gz?dl=0
Please confirm with 
R CMD INSTALL sesame_1.8.8.tar.gz

**Comment by jcm6t** [comment]
Closing this now.
